src/index.ts

```diff
@@ -14,7 +14,7 @@ export { default as Collapse } from './components/Collapse';
 export { default as DeletablePill } from './components/DeletablePill';
 export { default as Dropdown } from './components/Dropdown';
 export { default as EasyDropdown } from './components/EasyDropdown';
-export { default as EasyPill } from './components/EasyPill';
+export { default as EasyPill } from './components/EasyPill/EasyPill';
```

Comment:
You shouldn't need the extra `/EasyPill`
The `EasyPill/index` file re-exports the default from `./components/EasyPill`